modules/package.nix

```diff
@@ -305,6 +302,10 @@ in {
       default = pkgs.fetchurl { url = "mirror://hackage/${config.name}.tar.gz"; inherit (config) sha256; };
       defaultText = "pkgs.fetchurl { url = \"mirror://hackage/\${config.name}.tar.gz\"; inherit (config) sha256; };";
     };
+    package-description = mkOption {
```

Comment:
Doc? I know none of these options have doc, but I always wish they did, we could include it when we add options!

Reply:
I'm honestly not sure what this is. It's the _text_ of the cabal file, right?

Reply:
That's correct, and yes, I will add some documentation.